src/main/java/com/puppycrawl/tools/checkstyle/checks/indentation/AbstractExpressionHandler.java

```diff
@@ -414,7 +414,18 @@ protected final void checkExpressionSubtree(
      * @return          the first line of expression
      */
     protected static int getFirstLine(DetailAST tree) {
-        return getFirstAst(tree, tree).getLineNo();
+        return getFirstAstNode(tree).getLineNo();
+    }
+
+    /**
+     * Get the first ast for given expression.
+     *
+     * @param ast         the abstract syntax tree for which the starting ast is to be found
+     *
+     * @return the first ast of the expression
+     */
+    protected static DetailAST getFirstAstNode(DetailAST ast) {
```

Comment:
This change is refractors in indentation code and doesn't much related to this Issue.
Should I make a new commit for this?